cfme/tests/configure/test_log_depot_operation.py

```diff
@@ -235,9 +235,7 @@ def depot_configured(request, depot_type, depot_machine_folder, depot_credential
 @pytest.mark.tier(3)
 @pytest.mark.nondestructive
 @pytest.mark.meta(blockers=[BZ(1341502, unblock=lambda depot_type: depot_type != "anon_ftp",
-                            forced_streams=["5.6", "upstream"]),
-                            BZ(1380781, forced_streams=["5.7"])
-                            ]
+                            forced_streams=["5.6", "upstream"])]
```

Comment:
@psav this line related to another bug(BZ(1341502)). I just remove one bug, one still present